lib/__testfixtures__/get-set.input.js

```diff
@@ -0,0 +1,5 @@
+const arr = [1, 2, 3];
+arr.length = 5;
+const b = arr.length;
+const u = new URL();
+const c = u.pathname && 5;
```

Comment:
Can you test assigning to url.pathname? There are no tests for sets being rewritten anymore.